tests/tests/large_files_tests.robot

```diff
@@ -8,37 +8,28 @@ Resource            src/page_objects/keywords/common.robot
 Resource            src/page_objects/keywords/home_page.robot
 Resource            src/page_objects/keywords/ubuntu.robot
 Resource            src/test_files/test_files.resource
-Resource            /tmp/TestArtifacts/large_test_files.resource
 Library             src/page_objects/libraries/browser_lib.py
 
-Suite Setup         Prepare Test Environment
-Test Setup          Prepare For Test
 Test Teardown       Clean After Test
 
 
-*** Variables ***
-${TEST_FILE}                    ${file_1}
-${EXTRACTION_TIMEOUT}           ${file_1}[extraction_time]
-${PATH_TO_PATTERN_FOLDER}       ${file_1}[pattern_path]
-
-
 *** Test Cases ***
 Extract and validate a large test file
     [Documentation]    Extract and validate a large test file
     [Tags]    daily    regression    large
-    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${TEST_FILE}[archive_name].zip
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${large_file_15}[archive_name].zip
     # Add a file for extraxtion
     Click Add Files Button
-    Upload Test File    ${large_files_upload_path}${TEST_FILE}[name]
+    Upload Test File    ${input_test_files_path}${large_file_15}[name]
     # Load the file
     Click Load Button
-    Check If Log Console Contains    Opening "${TEST_FILE}[name]"
-    Validate Output Description    ${TEST_FILE}[archive_name]
-    Validate Output Archive File Size    ${TEST_FILE}[archive_size_bytes]
-    Validate Output Archive Files Number    ${TEST_FILE}[files_in_archive]
+    Check If Log Console Contains    Opening "${large_file_15}[name]"
+    Validate Output Description    ${large_file_15}[archive_name]
+    Validate Output Archive File Size    ${large_file_15}[archive_size_bytes]
+    Validate Output Archive Files Number    ${large_file_15}[files_in_archive]
     Wait Until Page Does Not Contain Element    ${ExtractAndSaveDisabledButton}
     # Extract and create a ZIP file
-    Click Extract And Save Button    ${EXTRACTION_TIMEOUT}
+    Click Extract And Save Button    ${large_file_15}[extraction_time]
```

Comment:
extraction_time not defined in test_files.resource

Reply:
fixed